sdk/eventgrid/azure-messaging-eventgrid/swagger/README.md

```diff
@@ -200,30 +201,29 @@ import java.util.stream.Collectors;
  */
 public class EventGridCustomization extends Customization {
 
-    private static final String SYSTEM_EVENT_CLASS_HEADER = "// Copyright (c) Microsoft Corporation. All rights reserved.\n" +
-        "// Licensed under the MIT License.\n" +
-        "\n" +
-        "package com.azure.messaging.eventgrid.implementation;\n\n";
-
-    private static final String CLASS_DEF = "/**\n" +
-        " * This class contains a number of constants that correspond to the value of {@code eventType} of {@link EventGridEvent}s\n" +
-        " * and {@code type} of {@link CloudEvent}s, when the event originated from an Azure service. This list should be\n" +
-        " * updated with all the service event strings. It also contains a mapping from each service event string to the\n" +
-        " * model class that the event string corresponds to in the {@code data} field, which is used to automatically deserialize\n" +
-        " * system events by their known string.\n" +
-        " */\n" +
-        "public final class SystemEventMappingNames {\n";
-
-    private static final String PRIVATE_CTOR = "/**\n" +
-        "     * Get a mapping of all the system event type strings to their respective class. This is used by default in\n" +
-        "     * the {@link EventGridEvent} and {@link CloudEvent} classes.\n" +
-        "     * @return a mapping of all the system event strings to system event objects.\n" +
-        "     */\n" +
-        "    public static Map<String, Class<?>> getSystemEventMappings() {\n" +
-        "        return Collections.unmodifiableMap(SYSTEM_EVENT_MAPPINGS);\n" +
-        "    }\n" +
-        "\n" +
-        "    private SystemEventMappingNames() { \n" +
+    private static final String SYSTEM_EVENT_CLASS_HEADER = "// Copyright (c) Microsoft Corporation. All rights reserved." + System.lineSeparator() +
```

Comment:
If this customization code is hard to maintain in a README, it can be moved to its own .java file like [storage](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-blob/swagger/src/main/java/BlobStorageCustomization.java) does.

Reply:
Yeah that's the next step. Alan suggested this also. I didn't want to wrap that into this update.